charts/karpenter/values.yaml

```diff
@@ -172,6 +172,8 @@ settings:
     # -- Cluster endpoint. If not set, will be discovered during startup (EKS only)
     clusterEndpoint: ""
     # -- The default instance profile to use when launching nodes
+    clusterServiceIpv4Cidr: ""
```

Comment:
Is it possible to discover this value from the cluster rather than specifying it within the `karpenter-global-settings`?

Reply:
Yes its possible. Currently it is discovering the value from the EKS cluster if the value is not specified. Perhaps we can skip specifying it in `karpenter-global-settings` altogether?